visual-inspection/inspection-backend/inspection/api.py

```diff
@@ -34,10 +36,12 @@ def method_must_be_available(cls, v):
             raise ValueError(f"{v} is not an available explanation method")
         return v
 
-
+#Nader We shoud add a top_index here to indey the top labels
 @api.post("/explain")
 def explain_classification(file: UploadFile = File(...),
                            method: Optional[str] = Form(None),
+                           index_of_label_to_explain: Optional[int] = Form(None),
+                           positive_only_parameter:  Optional[bool] = Form(None),
```

Comment:
See above: Please do not add settings as additional form fields, but use the `settings` field to provide specific settings.